streaming_timeseries/anaconda-project.yml

```
@@ -0,0 +1,67 @@
+name: streaming_timeseries
+description: Streaming display of multichannel timeseries data.
+
+examples_config:
+  created: 2024-12-06
+  maintainers:
+  - "droumis"
+  labels:
+  - "panel"
+  - "holoviews"
+  categories:
```

I've recently updated all the labels and added `bokeh` when hvplot/holoviews generate bokeh plots.

```suggestion
  - "holoviews"
  - "bokeh"
  categories:
```